pyscript.core/src/toml.js

```diff
@@ -0,0 +1,4 @@
+// https://cdn.jsdelivr.net/npm/@webreflection/toml-j0.4/toml.js
```

Comment:
Is there value in noting the specific release of toml-j0.4 here, just in case we (you) ever do an update/release of that project and we want to be able to tell which version is included here? `1.1.3` it lookslike?

Reply:
AFAIK nobody is planning to maintain or change that project so I might indeed jut create yet another automated artifact that grabs that from latest version (expected to be always the same) and place it in the src folder before building it.

Or maybe I can just add it as npm dependency and still automate that process ... this actually might be the best way to go:

  * the version is embedded in the package.json
  * the automation simply reflects that

Good hint!

Reply:
**done**!